app/scripts/modules/core/src/securityGroup/securityGroupReader.service.ts

```diff
@@ -117,6 +117,16 @@ export class SecurityGroupReader {
     }
   }
 
+  private static sortUsages(securityGroup: ISecurityGroup): void {
+    if (!securityGroup.usages) {
+      return;
+    }
+    // reverse sort - it's gross but keeps versions mostly sorted in the chronological order
+    securityGroup.usages.serverGroups.sort((a, b) => b.name.localeCompare(a.name));
+    // reverse sort - gross but what we are doing now and consistent with the server groups
+    securityGroup.usages.loadBalancers.sort((a, b) => b.name.localeCompare(a.name));
```

Comment:
ugh. i always forget we don't stick a created timestamp on these objects. probably never fixed it because it only stops working for that brief period of time where your sequence numbers wrap around 😢